src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -150,6 +197,10 @@ export function transform(formConfig, form) {
     // if there is at least one treatment to send
     ...(treatments.length && { treatments }),
     ...(attachments.length && { attachments }),
+    form4142: {
+      limitedConsent: gatherLimitedConsentText(disabilities),
```

Comment:
We probably don't need to send this if the string's empty, but not sure if this even matters or if the implementation is already agreed upon with vets-api.

Reply:
The api can handle a blank string/won't populate anything on the form. So there isn't any harm in returning an empty string, or the need to conditionally check for an empty string.